test/jit/test_backends.py

```diff
@@ -0,0 +1,63 @@
+import os
+import sys
+import unittest
+
+import torch
+import torch._C
+from torch.testing import FileCheck
+from torch._six import PY2
+
+# Make the helper files in test/ importable
+pytorch_test_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+sys.path.append(pytorch_test_dir)
+from torch.testing._internal.jit_utils import JitTestCase
+
+if __name__ == '__main__':
+    raise RuntimeError("This test file is not meant to be run directly, use:\n\n"
+                       "\tpython test/test_jit.py TESTNAME\n\n"
+                       "instead.")
+
+
+def to_test_backend(module, method_compile_spec):
+    return torch._C._jit_to_test_backend(module, {"forward": method_compile_spec})
+
+
+class MyModule(torch.nn.Module):
+    def __init__(self):
+        super(MyModule, self).__init__()
+
+    def forward(self, x, h):
+        new_h = x + h
+        return new_h
+
+    def __setstate__(self, state):
+        pass
+
+    def __getstate__(self):
+        return None
+
+
+class TestBackends(JitTestCase):
+    def test_simple(self):
+        module = MyModule()
+        scripted_module = torch.jit.script(MyModule())
+
+        # Test compile.
+        lowered_module = to_test_backend(scripted_module._c, {"key": "value"})
+
+        # Test execute against Python and JIT.
+        input = torch.randn(5)
+        
+        python_output = module(input, input)
+        jit_output = scripted_module(input, input)
+        backend_output = lowered_module.forward(input, input)
+
+        # The answers returned by Python, JIT and to_backend should all match.
+        self.assertEqual(python_output, backend_output)
+        self.assertEqual(jit_output, backend_output)
+
+        # Test save and load.
+        lowered_module.save("lowered.pt")
+        # loaded_module = torch.jit.load("lowered.pt")
```

Comment:
So serialization works now, but deserialization doesn't work because `Any` types are not deserializable. cc: @suo